lib/view/monthlyEventView.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:flutter/material.dart';
```

Comment:
Is this just the old monthlyeventview?

Reply:
yep, the old one was misspelled "monthlyEvenView"